Consul/Consul.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <VersionPrefix>1.6.1.1</VersionPrefix>
+    <VersionPrefix>1.6.1.3</VersionPrefix>
```

Comment:
I would suggest using version `1.7.0`:
- Three-part version number is more common versioning scheme for NuGet, https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-basics
- According to the semantic versioning rules (https://semver.org/spec/v2.0.0.html) you should bump the `Minor` version:
```
...
- Bug fixes not affecting the API increment the patch version,
- Backwards compatible API additions/changes increment the minor version
- Backwards incompatible API changes increment the major version.
...
```

Reply:
I was also wondering a bit about the 4-digits notation, I only wanted to stay on the same pattern. Now if we think about three digits its for sure a `1.7.0`.

Let's follow the NuGet paradigm. I'll update to `1.7.0`